lib/bundler/postit_trampoline.rb

```diff
@@ -20,7 +20,10 @@ module BundlerVendoredPostIt; end
   if !version.satisfied_by?(installed_version)
     begin
       installer = BundlerVendoredPostIt::PostIt::Installer.new(version)
-      installer.install!
+      unless installer.installed?
+        warn "Installing locked Bundler version #{version}"
```

Comment:
We realize that the Gem::Version uses `=` to mean "an exact matching version", but most people don't, and I think end users will generally be confused by it.

Let's remove the `= ` from the beginning of the version, and add `...` to the end to indicate that it will take a moment to happen. So the message should look like: `Installing locked Bundler version 1.12.6...`